### ISSUE TITLE
Kodi Library verification

### ISSUE THREAD

**Issue by teotikalki** [barbequesauce/Watcher3#87]

**Comment by barbequesauce** [comment]
Hi - did you see the Kodi plugins in the plugins repo, as described in the wiki?

**Comment by scambra** [comment]
I'm not sure what's this request about. You can import your Kodi Library, then to keep both in sync you should download everything from watcher, and you can use plugins to start library update in kodi. If you add movies to kodi manually, you can import again, only movies not in watcher db will be imported.

**Comment by teotikalki** [comment]
Yeah, sorry. I deleted my issue right after posting because I realized I had to do a lot more on my end: watcher is installed on a vm on 172.16.*.*  and kodi is on my desktop on 10.*.*.*; there is currently no headless kodi to attach to Watcher.

After Watcher is aware of Kodi I'll be able to actually evaluate usage.
That being said: I add all items to Kodi manually (this one time this BDMV scanned in as almost a hundred copies of a short film or something and it was faster to remake my library from scratch...). I also like to sort series (so there's directory @Marvel.Cinematic.Universe, and inside is a directory @Iron.Man, inside of which are actual movies... etc.) I've had annoyances like the directory showing up as the movie when the series shares a name with (usually) the first episode (Star Wars, The Mummy, etc). All in all I'm just more comfortable knowing exactly what changes are being made because I am discretely making them. Watcher is, to me, mostly a pretty wishlist with metadata. I have yet to use it for actually sourcing media. (This is actually a really big deal to me this week: Gnote lost my 'Movies-Wishlist' note and several years worth of random 'oh yeah, that movie' moments that had been compiled are now gone. :( )
